tools/interactive/interactivetool_ilastik.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="interactive_tool_ilastik" tool_type="interactive" name="Ilastik" version="@VERSION@" profile="23.0">
+    <description>interactive learning and segmentation toolkit</description>
+    <macros>
+        <token name="@VERSION@">1.4.0</token>
+    </macros>
+    <requirements>
+        <container type="docker">quay.io/galaxy/ilastik:@VERSION@</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="Ilastik" requires_domain="True">
+            <port>5800</port>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code">
+    <![CDATA[
+        export HOME=\$PWD &&
+        ## Make a copy of the existing project if exists
+        #if str($input_type.existing) == "existing":
+            cp '$input_type.project' ./MyProject.ilp &&
+        #end if
+
+        ## Link input images to current working directory
+        #for input in $infiles:
+            ln -s '$input' ./'$input.name' &&
```

Comment:
If the history name of two dataset is the same this will crash. Maybe we can detect this, and prefix in this case?
We should also use '$input.element_identifier'

Reply:
Does Ilastik itself tolerate 2 input files with the same name during import or some other step? If so, maybe we should error out the job if names are not unique?

Reply:
Good catch @bgruening . Thanks. Ilastik does not see the $infiles before the user add them to the project so having identical names is not an issue (for example, it is possible to have identical names in different directories I think).

Reply:
As this interactive tool also allow to modify an existing project I would go for a tool failure when identifiers are identical. But I will make a better error handling.